docs/process/graduation.md

```diff
@@ -22,8 +22,10 @@ This document contains a checklist for the requirements to migrate a component f
   - [ ] Use `dep:` for enabling dependencies
 - [ ] The crate should be fully documented
   - [ ] Every exported function should have docs coverage
-  - [ ] There should be a crate-level example that illustrates a common use case for the component
-  - [ ] All options and conditional code paths should have a corresponding docs test
+  - [ ] There should be a crate-level example that illustrates a common use case for the component with the heading `# Examples`
+  - [ ] All options and conditional code paths should have a corresponding docs test with the heading `# Examples`
+  - [ ] All functions that are conditional on a Cargo feature should say so (last line before `# Examples`): ```✨ ***Enabled with the `"alloc"` Cargo feature.***```
```

Comment:
```suggestion
  - [ ] All functions that are conditional on a Cargo feature should say so (last line before `# Examples`): ```✨ **Enabled with the `"alloc"` Cargo feature.**```
```

Reply:
My proposal is ***bold italic***, not just **bold**.

Reply:
Yes, my proposal is just italic

Reply:
Actually that's just one asterisk.

Reply:
I think it works well with the emoji
<img width="639" alt="image" src="https://github.com/unicode-org/icu4x/assets/4706271/155baf62-060d-41ab-8b37-3863e691f247">

Reply:
Ok, one asterisk (italic).

Reply:
We might also want to drop the `"`s:
<img width="625" alt="image" src="https://github.com/unicode-org/icu4x/assets/4706271/9f2b323d-a432-4e37-8762-6f0698d852a1">

Reply:
Updated to plain italics